database.py

```diff
@@ -3,8 +3,12 @@
 import yaml
 
 
-with open('secrets.yaml') as file:
-    secrets = yaml.load(file)
-    engine = create_engine(secrets['sqlalchemy_connection_string'], echo=True)
+def get_connection(is_debug):
+    with open('secrets.yaml') as file:
+        secrets = yaml.load(file)
+        if is_debug:
+            engine = create_engine(secrets['sqlalchemy_connection_string_qa'], echo=True)
```

Comment:
I've seen other applications use different YAML files to separate environments (i.e. testing.yaml, production.yaml) but this works too